exotica_core/include/exotica_core/kinematic_tree.h

```diff
@@ -145,9 +154,16 @@ class KinematicTree : public Uncopyable
     const Eigen::MatrixXd& GetJointLimits() const { return joint_limits_; }
     void SetJointLimitsLower(Eigen::VectorXdRefConst lower_in);
     void SetJointLimitsUpper(Eigen::VectorXdRefConst upper_in);
+    void SetJointLimitsVelocity(Eigen::VectorXdRefConst velocity_in);
+    void SetJointLimitsAcceleration(Eigen::VectorXdRefConst acceleration_in);
+    void SetFloatingBaseLimitsPosXYZEulerZYX(const std::vector<double>& lower, const std::vector<double>& upper, const std::vector<double>& velocity, const std::vector<double>& acceleration);
+    void SetPlanarBaseLimitsPosXYEulerZ(const std::vector<double>& lower, const std::vector<double>& upper, const std::vector<double>& velocity, const std::vector<double>& acceleration);
     void SetFloatingBaseLimitsPosXYZEulerZYX(const std::vector<double>& lower, const std::vector<double>& upper);
     void SetPlanarBaseLimitsPosXYEulerZ(const std::vector<double>& lower, const std::vector<double>& upper);
     std::map<std::string, std::vector<double>> GetUsedJointLimits() const;
+    bool GetHasAccelerationLimit() const { return has_acceleration_limit; }
```

Comment:
Rename to ```HasAccelerationLimit()```

Reply:
Not get_HasAccelerationLimit? See conversation with @wxmerkt here: https://github.com/ipab-slmc/exotica/pull/715#discussion_r424650911